lib/generator.js

```diff
@@ -10,170 +10,169 @@ import { promptText } from "./prompt.js";
-    const prompt = promptText(summary, diff, this.config);
-    try {
-      const model = this.ai.getGenerativeModel({ model: this.config.model });
-      const result = await model.generateContent(prompt);
-      const response = result.response;
-      let message = response.text();
-
-      // Clean up response
-      message = message
-        .replace(/^```/, "")
-        .replace(/Commit Message:/i, "")
-        .trim();
-
-      // Validate format
-      if (
-        !/^(feat|fix|docs|style|refactor|test|chore|perf|build|ci|revert)(\(.+\))?: .+/.test(
-          message
-        )
-      ) {
-        if (retries < this.config.maxRetries) {
-          return this.generateCommitMessage(diff, summary, retries + 1);
-        }
-        console.warn(
-          chalk.yellow("Could not generate conventional commit format")
-        );
-      }
-      if (this.config.prefix) {
-        message = `${this.config.prefix} : ${message}`;
-      }
-
-      return message;
-    } catch (error) {
-      if (retries < this.config.maxRetries) {
-        return this.generateCommitMessage(diff, summary, retries + 1);
-      }
-      throw new Error(`Failed to generate message: ${error.message}`);
-    }
-  }
-
-  async createCommit(message) {
-    try {
-      const escapedMessage = message.replace(/"/g, '\\"');
-      const { stdout, stderr } = await execAsync(
-        `git commit -m "${escapedMessage}"`
-      );
-      if (stderr) console.warn(chalk.yellow(stderr));
-      return true;
-    } catch (error) {
-      throw new Error(`Commit failed: ${error.stderr || error.message}`);
-    }
-  }
-
-  displayMessage(message) {
-    const wrapped = wrap(message, 60);
-    const box = boxen(wrapped, {
-      padding: 1,
-      margin: 1,
-      borderStyle: "round",
-      borderColor: "blue",
-      backgroundColor: "#222222"
-    });
-    console.log(box);
-  }
-
-  async run() {
-    console.log(chalk.blue("\n🤖 AI Commit Message Generator\n"));
-
-    // Verify git repo
-    try {
-      await execAsync("git rev-parse --is-inside-work-tree");
-    } catch {
-      throw new Error("Not in a Git repository");
-    }
-
-    // Get changes
-    const [diff, summary] = await Promise.all([
-      this.getGitDiff(),
-      this.getDiffSummary()
-    ]);
-
-    if (!diff.trim()) {
-      console.log(chalk.yellow("No staged changes found."));
-      console.log(`Use ${chalk.cyan("git add")} to stage changes first.`);
-      return;
-    }
-
-    // Generate message
-    const message = await this.generateCommitMessage(diff, summary);
-    this.displayMessage(message);
-
-    // User interaction
-    const response = await prompts([
-      {
-        type: "select",
-        name: "action",
-        message: "What would you like to do?",
-        choices: [
-          { title: "✅ Commit with this message", value: "commit" },
-          { title: "✏️  Edit message", value: "edit" },
-          { title: "🔄 Regenerate message", value: "regenerate" },
```

Comment:
Replaced:
'Edit message' with 'Edit the message'
'Regenerate message' with 'Regenerate the message'